src/index.html

```diff
@@ -1,27 +1,485 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
-    <title>Catalog</title>
-    <link
-      rel="preconnect"
-      href="https://fonts.gstatic.com"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      rel="stylesheet"
-      href="styles/index.scss"
-    />
-  </head>
-
-  <body>
-    <h1>Catalog</h1>
-  </body>
+<head>
+  <meta charset="UTF-8" />
+  <meta
+    name="viewport"
+    content="width=device-width, initial-scale=1.0"
+  />
+  <title>Catalog</title>
+  <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+  />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet"
+  />
+  <link
+    rel="stylesheet"
+    href="styles/index.scss"
+  />
+</head>
+
+<body>
+<header class="menu">
+  <div class="item logo">
+    <a href="#">
+      <img
+        src="images/logo.png"
+        alt="Logo"
+      />
+    </a>
+  </div>
+
+  <nav>
+    <ul class="menu-items">
+      <li class="item">
+        <a class="list is-active" href="#">
+          Apple
+        </a>
+      </li>
+
+      <li class="item item-3">
+        <a class="list" href="#">
+          samsung
+        </a>
+      </li>
+
+      <li class="item item-4">
+        <a class="list" href="#">
+          smartphones
+        </a>
+      </li>
+
+      <li class="item item-5">
+        <a class="list list--item-5" data-qa="nav-hover" href="#">
+          Laptops & Computers
+        </a>
+      </li>
+
+      <li class="item item-6">
+        <a class="list" href="#">
+          Gadgets
+        </a>
+      </li>
+
+      <li class="item item-7">
+        <a class="list" href="#">
+          Tablets
+        </a>
+      </li>
+
+      <li class="item item-8">
+        <a class="list" href="#">
+          Photo
+        </a>
+      </li>
+
+      <li class="item item-9">
+        <a class="list" href="#">
+          Video
+        </a>
+      </li>
+    </ul>
+  </nav>
+</header>
+
+<main class="main">
+  <section class="product-card" data-qa=«card»>
```

Comment:
The data attribute `data-qa=«card»` uses non-standard quotation marks. It should be changed to standard double quotes like `data-qa="card"`.